### PR TITLE
Enable JupyterLab web access for both host and docker

### PR DESCRIPTION
**Pull request by HongW2019**

### CODE REVIEW

File: runtime/spark/docker/Dockerfile [hunk 1]
```diff
@@ -48,20 +48,4 @@ COPY requirements.txt /tmp/requirements.txt
...
-RUN mkdir -p /home/cloudtik/jupyter && echo Y|jupyter notebook --generate-config
-RUN sed -i  "1 ic.NotebookApp.ip = '*'" ~/.jupyter/jupyter_notebook_config.py && \
-    sed -i  '1 ic.NotebookApp.open_browser = False' ~/.jupyter/jupyter_notebook_config.py && \
-    sed -i  "1 ic.NotebookApp.notebook_dir = '/home/cloudtik/jupyter'" ~/.jupyter/jupyter_notebook_config.py && \
-    # set default password(cloudtik) for jupyter lab
-    sed -i  "1 ic.NotebookApp.password = 'argon2:\$argon2id\$v=19\$m=10240,t=10,p=8\$Y+sBd6UhAyKNsI+/mHsy9g\$WzJsUujSzmotUkblSTpMwCFoOBVSwm7S5oOPzpC+tz8'" ~/.jupyter/jupyter_notebook_config.py && \
-    sed -i  '1 ic.NotebookApp.allow_root = True'   ~/.jupyter/jupyter_notebook_config.py
-
-# jupyter kernel
-RUN python -m spylon_kernel install --user
```
Comment (jerrychenhf) [inline runtime/spark/docker/Dockerfile:66]: To install something in docker image has advantage to save cluster startup time.
Can we figure out some clean way that we install something in the docker and install through spark installation only when it is not installed (for example host mode)?

For configuring, we can do always do it if it is not time consuming.

Comment (HongW2019) [reply runtime/spark/docker/Dockerfile:66]: Done.

File: runtime/spark/scripts/install.sh [hunk 2]
```diff
@@ -46,6 +46,20 @@ if [ ! -d "${SPARK_HOME}" ]; then
   echo "export PATH=\$SPARK_HOME/bin:\$PATH" >> ${USER_HOME}/.bashrc
 fi
 
+# Install Jupyter and spylon-kernel for Spark
+if ! type jupyter >/dev/null 2>&1; then
+  echo "Install JupyterLab..."
+  pip3 install jupyterlab
```
Comment (jerrychenhf) [inline runtime/spark/scripts/install.sh:52]: Is there any specific reason that we use pip3 specifically? Use "pip" not working?

Comment (HongW2019) [reply runtime/spark/scripts/install.sh]: Change it to `pip`.

**Pull request merged**